packages/broker/src/ConfigWizard.ts

```diff
@@ -82,200 +87,168 @@ const DefaultConfig: Config = {
+    },
 }
 
-export class ConfigWizard{
-    config: Config
-    // required to mock results on tests
-    inquirer: inquirer.Inquirer = inquirer
-
-    constructor() {
-        this.config = DefaultConfig
-    }
-
-    private async inquirerSinglePrompt(prompt: inquirer.Question | inquirer.ListQuestion | inquirer.CheckboxQuestion) {
-        const answers = await this.inquirer.prompt([prompt])
-        return answers[prompt.name!]
-    }
-
-    private async promptNumberWithDefault(prompt: inquirer.InputQuestion, defaultValue: number): Promise<number>{
-        const valueString = await this.inquirerSinglePrompt(prompt)
-        const value = valueString === '' ? defaultValue: parseInt(valueString)
-
-        if (Number.isNaN(value) || !Number.isInteger(value)) {
-            logger.warn(`Non-numeric value [${valueString}] provided`)
-            return await this.promptNumberWithDefault(prompt, defaultValue)
-        }
+const promptValues = {
+    generate: 'Generate',
+    import: 'Import'
+}
 
-        if (value < 1024 || value > 49151) {
-            logger.warn(`Out of range port [${value}] provided (valid range 1024-49151)`)
-            return await this.promptNumberWithDefault(prompt, defaultValue)
+let prompts: Array<inquirer.Question | inquirer.ListQuestion | inquirer.CheckboxQuestion> = [
+    {
+        type: 'list',
+        name:'generateOrImportEthereumPrivateKey',
+        message: 'Do you want to generate a new Ethereum private key or import an existing one?',
+        choices: [promptValues.generate, promptValues.import]
+    },
+    {
+        type: 'input',
+        name:'importPrivateKey',
+        message: 'Please provide the private key to import',
+        when: (answers: inquirer.Answers) => {
+            return answers.generateOrImportEthereumPrivateKey === promptValues.import
+        },
+        validate: (input: string): string | boolean => {
+            try {
+                new Wallet(input)
+                return true
+            } catch (e) {
+                if (e.message.includes('invalid hexlify value')){
+                    return `Invalid privateKey provided for import: ${input}`
+                } else {
+                    return e.message
+                }
+            }
         }
-
-        return value
     }
+]
 
-    private async promptExistingPathWithDefault(prompt: inquirer.InputQuestion, defaultValue: string): Promise<string>{
-        const returnedValue = await this.inquirerSinglePrompt(prompt)
-        // defaultValue always exists at this point
-        if (returnedValue === '') {
-            return defaultValue
-        }
-
-        if (!existsSync(returnedValue)) {
-            logger.warn(`Path [${returnedValue}] does not exist`)
-            return await this.promptExistingPathWithDefault(prompt, defaultValue)
-        }
+const PLUGIN_TEMPLATES = {
+    websocket: { port: DEFAULT_WS_PORT },
+    mqtt: { port: DEFAULT_MQTT_PORT },
+    publishHttp: { port: DEFAULT_HTTP_PORT }
```

Comment:
The `publishHttp` plugin doesn't have a `port` property in the plugin configuration. It uses whatever port is configured to `httpServer`. So we should maybe set the value to httpServer.port (line 62) if user wants to override the port. 

And if user doesn't override some port (a plugin port or `httpServer` port) it would be maybe easiest to leave the configuration out. For HTTP server we can leave the whole `httpServer` block out in that case.